tests/models/idefics/test_processor_idefics.py

```diff
@@ -208,3 +217,164 @@ def test_model_input_names(self):
 
         # For now the processor supports only ['pixel_values', 'input_ids', 'attention_mask']
         self.assertSetEqual(set(inputs.keys()), set(self.input_keys))
+
+    @require_vision
+    @require_torch
+    def test_tokenizer_defaults_preserved_by_kwargs(self):
+        if "image_processor" not in self.processor_class.attributes:
```

Comment:
imo we don't have to overwrite all tests for idefics, seems quite similar to the general test from mixin except for ` padding="max_length"`. We can and maybe should indicate padding="max_length" in mixin tests, cause we can't assume all tokenizer will default to "max_length" padding